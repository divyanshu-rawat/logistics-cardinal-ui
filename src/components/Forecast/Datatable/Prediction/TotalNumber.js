import React from 'react';

const TotalNumber = () => <div />;

export default TotalNumber;
