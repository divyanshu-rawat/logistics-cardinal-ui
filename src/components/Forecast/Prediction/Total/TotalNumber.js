import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const TotalNumberStyled = styled.span`
  color: white;
  position: absolute;
  top: 50%;
  left: 15px;
  transform: translateY(-50%);
  z-index: 2;
`;

const TotalNumber = ({ total }) => (
  <TotalNumberStyled>{total}</TotalNumberStyled>
);

TotalNumber.propTypes = {
  total: PropTypes.number.isRequired,
};

export default TotalNumber;
