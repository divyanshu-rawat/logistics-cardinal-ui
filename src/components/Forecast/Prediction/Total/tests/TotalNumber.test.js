import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TotalNumber from '../TotalNumber';

const renderedComponent = () => shallow(<TotalNumber total={100} />);

describe('<TotalNumber />', () => {
  it('should render the component with the correct number', () => {
    const tree = toJson(renderedComponent());
    expect(tree).toMatchSnapshot();
  });
});
