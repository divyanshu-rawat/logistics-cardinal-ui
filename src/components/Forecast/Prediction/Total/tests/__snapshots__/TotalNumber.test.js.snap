// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TotalNumber /> should render the component with the correct number 1`] = `
<TotalNumber__TotalNumberStyled>
  100
</TotalNumber__TotalNumberStyled>
`;
